docarray/array/storage/milvus/find.py

```diff
@@ -0,0 +1,56 @@
+from typing import TYPE_CHECKING, TypeVar, List, Union, Optional, Dict, Sequence
+
+if TYPE_CHECKING:
+    import numpy as np
+    import tensorflow
+    import torch
+
+    # Define the expected input type that your ANN search supports
+    MilvusArrayType = TypeVar(
+        'MilvusArrayType',
+        np.ndarray,
+        tensorflow.Tensor,
+        torch.Tensor,
+        Sequence[float],
+    )
+    from docarray import Document, DocumentArray
+
+
+class FindMixin:
+    def _find(
+        self,
+        query: 'MilvusArrayType',
+        limit: int = 10,
+        filter: Optional[Dict] = None,
+        param=None,
+        **kwargs
+    ) -> Union['DocumentArray', List['DocumentArray']]:
```

Comment:
the return type is always List['DocumentArray'] in the backend level, the parent Find mixin will do auto boxing

Reply:
I was going by the type hints provided in the backend implementation docs, we need to change it there then:
![image](https://user-images.githubusercontent.com/44071807/199949245-6e9bd5ca-51c2-450a-a1ef-2bd42f851305.png)